### ISSUE TITLE
HTP nc1669 and phenotypes etc for some ncRNAs PMID:36259651

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3450]
Hi @manulera 
Once you have done Maria's paper, could you make an import table for the phenotypes for the ncRNAs from this publication:
https://pubmed.ncbi.nlm.nih.gov/36259651/
Structure-based screening for functional non-coding RNAs in fission yeast identifies a factor repressing untimely initiation of sexual differentiation
It has some screensa and partially characterised ncRNAs.

@afg RNACentral might be interested in this publication, there are a small number of the intergenic RNAs that are conserved in other Schizos which might be amenable to R2D2? Maybe somebody would like to take a look. On another note we have a few screens coming through that contain phenotypic data for  quite a lot of the ncRNAs (this one, plus 2 much larger ones). We can supply this data to RNAcentral in this format: https://www.pombase.org/downloads/phenotype-annotations
We would supply a file with just the RNA data for RNACentral.

**Comment by ValWood** [comment]
@manulera  this is to make sure we have all of the 'named' non coding RNAs  with some functional annotation annotated.  nc1669 is the main point of interest for this purpose.

**Comment by manulera** [comment]
Ok! Noted!

**Comment by manulera** [comment]
Hi @ValWood I looked into the paper, and the supplementary data has no table of scores of the phenotypes, just raw plate scans in a pdf in the supplementary data, unless I am missing something.

We could write to them, but if they are not reporting numbers in the figures or the supplementary, I doubt they quantified it. I looked at the figures by eye and it seems the only obvious phenotypes they got are the ones reported in the text. No phenotypes in mating efficiency when waiting 24 hours either (Fig. S8)
